internal/stack/stacks.go

```diff
@@ -94,11 +101,10 @@ func getStacks(all bool) []Stack {
 				state:     goState,
 				fullStack: &bytes.Buffer{},
 			}
-			isFirstLine = true
 		}
 		curStack.fullStack.WriteString(line)
-		if !isFirstLine && curStack.firstFunction == "" {
-			curStack.firstFunction = parseFirstFunc(line)
+		if f := parseFunc(line); f != "" {
+			curStack.functions = append(curStack.functions, f)
 		}
```

Comment:
this looks like it's trying to parse every line, should we instead parse only the lines known to be functions?

I think the format is:
```
<function>
<tab><file:line>
<function>
<tab><file:line>
...
```